.github/workflows/tests.yaml

```diff
@@ -25,30 +25,30 @@ jobs:
         postgis: [false]
         dev: [false]
         env:
-          - ci/travis/36-minimal.yaml
-          - ci/travis/38-no-optional-deps.yaml
-          - ci/travis/36-pd025.yaml
-          - ci/travis/37-latest-defaults.yaml
-          - ci/travis/37-latest-conda-forge.yaml
-          - ci/travis/38-latest-conda-forge.yaml
+          - ci/actions/36-minimal.yaml
```

Comment:
Can you maybe use `envs/` or something like that instead? That's a bit more neutral, and more explicit about what it is